backend/fleet_management/admin.py

```diff
@@ -126,3 +126,17 @@ class CustomUserAdmin(UserAdmin):
         ),
         (_("Important dates"), {"fields": ("last_seen", "last_login", "date_joined")}),
     )
+
+
+@admin.register(Refuel)
+class RefuelAdmin(admin.ModelAdmin):
+    list_filter = ("driver", "car")  # project, country?
```

Comment:
```suggestion
    list_filter = ("driver", "car", "project", CountryFilter)
```

AC3: There should be filtering by country, project, driver, and car enabled.